qdrant-landing/content/documentation/fastembed/fastembed-quickstart.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Quickstart" 
+weight: 2
+---
+
+# How to Generate Text Embedings with FastEmbed
+
+## Install FastEmbed
+```python
+pip install fastembed
+```
+Just for demo purposes, you will use Lists and NumPy to work with sample data.
```

Comment:
I am not sure whether this statement is required, because fastembed uses numpy all the time